docs/xml/enumerations/district_type.rst

```diff
@@ -9,41 +9,43 @@ DistrictType
 | Name                 | Description                                                                      |
 |                      |                                                                                  |
 +======================+==================================================================================+
-| city                 |                                                                                  |
+| city                 | A city.                                                                          |
 +----------------------+----------------------------------------------------------------------------------+
-| city-council         |                                                                                  |
+| city-council         | A city, town, or village council, either in its entirety or for a specific seat. |
```

Comment:
What does "in its entirety" mean here?

city-council means "a city council district," not "the city" (which would be city). At-large city council members are (by definition) elected citywide, so a contest for an at-large city council seat would be associated with the electoral district for the city, which would have type "city."